lib/home_page.dart

```diff
@@ -0,0 +1,341 @@
+                  itemBuilder: (_, i) {
+                    int a = 2 * i;
+                    int b = 2 * i + 1;
+                    return Row(
+                      children: [
+                        Container(
+                          padding: const EdgeInsets.only(bottom: 5),
+                          width: 170,
+                          height: 160,
+                          margin: const EdgeInsets.only(left: 20, bottom: 10),
+                          decoration: BoxDecoration(
+                              color: Colors.blueAccent,
+                              borderRadius: BorderRadius.circular(10),
+                              image: DecorationImage(
+                                  image: AssetImage(info[i]['img'])),
+                              boxShadow: [
+                                BoxShadow(
+                                    blurRadius: 3,
+                                    offset: const Offset(5, 5),
+                                    color: color.AppColor.gradientSecond
+                                        .withOpacity(0.1))
+                              ]),
+                          child: Center(
+                            child: Align(
+                              alignment: Alignment.bottomCenter,
+                              child: Text(
+                                info[a]['title'],
+                                style: const TextStyle(
+                                    fontSize: 20, color: Colors.white),
+                              ),
+                            ),
+                          ),
+                        ),
+                        Container(
+                          padding: const EdgeInsets.only(bottom: 5),
+                          width: 170,
+                          height: 160,
+                          margin: const EdgeInsets.only(left: 10, bottom: 10),
+                          decoration: BoxDecoration(
+                              color: Colors.blueAccent,
+                              borderRadius: BorderRadius.circular(10),
+                              image: DecorationImage(
+                                  image: AssetImage(info[b]['img'])),
+                              boxShadow: [
+                                BoxShadow(
+                                    blurRadius: 3,
+                                    offset: const Offset(5, 5),
+                                    color: color.AppColor.gradientSecond
+                                        .withOpacity(0.1))
+                              ]),
+                          child: Center(
+                            child: Align(
+                              alignment: Alignment.bottomCenter,
+                              child: Text(
+                                info[b]['title'],
+                                style: const TextStyle(
+                                    fontSize: 20, color: Colors.white),
```

Comment:
i think here it is better to give a smaller font size, example 18 or 16, it will look a better in a horizontal position